add support for simplified chinese 简体中文化

dtlnor opened the pull request:
WIP, any update is welcome
抛砖引玉，只做了粗略的翻译，还需要精细打磨，欢迎纠错~~（我翻不了那么多，求大佬）~~
update: I've finished all the translation, welcome for any improvement

tranzmatt committed:
Pass arguments from bash to python

dfaker committed:
add windows equivalent

MalumaDev committed:
init

zhengxiaoyao0716 committed:
reload javascript files when custom script bodies

MalumaDev committed:
fix to tokens lenght, addend embs generator, add new features to edit the embedding before the generation using text

MalumaDev committed:
Merge branch 'master' into test_resolve_conflicts

MalumaDev committed:
Update modules/ui.py

Co-authored-by: Víctor Gallego <vicgalle@ucm.es>

MalumaDev committed:
Update README.md

Co-authored-by: Víctor Gallego <vicgalle@ucm.es>

MalumaDev committed:
Update modules/aesthetic_clip.py

Co-authored-by: Víctor Gallego <vicgalle@ucm.es>

MalumaDev committed:
Update modules/ui.py

Co-authored-by: Víctor Gallego <vicgalle@ucm.es>

MalumaDev committed:
Update modules/ui.py

Co-authored-by: Víctor Gallego <vicgalle@ucm.es>

MalumaDev committed:
Update modules/ui.py

Co-authored-by: Víctor Gallego <vicgalle@ucm.es>

MalumaDev committed:
Add support to other img format, fixed dropbox update

MalumaDev committed:
Merge branch 'master' into test_resolve_conflicts

MalumaDev committed:
fixed dropbox update

MalumaDev committed:
Merge remote-tracking branch 'origin/test_resolve_conflicts' into test_resolve_conflicts

MalumaDev committed:
ui fix

MalumaDev committed:
ui fix

dfaker committed:
add update warning to launch.py

dfaker committed:
move update check to after dep installation

Mozoloa committed:
Better Bug report form

Mozoloa committed:
Delete .vscode directory

Mozoloa committed:
Cleaning & improvements

Mozoloa committed:
Fixes and adding step by step

Mozoloa committed:
fixing linebreak issue

Mozoloa committed:
Fixes and trying to make dropdown required

Mozoloa committed:
removing the required tag as it obviously doesn't work, adding a top description

Mozoloa committed:
Adding a confirmation checkbox that the user has checked the issues & commits before

Also small fixes

Mozoloa committed:
Fixed labels and created a brand new Feature Request yaml

Mozoloa committed:
Formatting the top description

Mozoloa committed:
Removed obsolete legacy Hlky description

MalumaDev committed:
ui fix, re organization of the code

MalumaDev committed:
Merge branch 'master' into test_resolve_conflicts

dfaker committed:
Make update check commandline option, give response on all paths.

dfaker committed:
add shared option for update check

MalumaDev committed:
Merge branch 'master' into test_resolve_conflicts

camenduru committed:
Remove duplicate artists.

ArcticFaded committed:
initial prototype by borrowing contracts

ArcticFaded committed:
pydantic instrumentation

ArcticFaded committed:
Update webui.py

ArcticFaded committed:
Update processing.py

ArcticFaded committed:
Update processing.py

ArcticFaded committed:
Update api.py

AUTOMATIC1111 committed:
add api() function to return webui() to how it was

AUTOMATIC1111 committed:
add missing requirement for api and fix some typos

Michoko92 committed:
Added dark mode switch

Launch the UI in dark mode with the --dark-mode switch

overra committed:
prevent highlighting/selecting image

w-e-w committed:
styles.csv encoding utf8 to utf-8-sig

utf-8-bom for better compatibility for some programs

trufty committed:
Disable auto weights swap with config option

Michoko92 committed:
Use of a --theme argument for more flexibility

Added possibility to set the theme (light or dark)

C43H66N12O12S2 committed:
use legacy attnblock

C43H66N12O12S2 committed:
delete xformers attnblock

simcop2387 committed:
Re-use webui fastapi application rather than requiring one or the other, not both.

simcop2387 committed:
Put API under /sdapi/ so that routing is simpler in the future.  This means that one could allow access to /sdapi/ but not the webui.

simcop2387 committed:
Add --nowebui as a means of disabling the webui and run on the other port

Mozoloa committed:
Improved the OS/Platforms field

guaneec committed:
Don't eat colons in booru tags

ArcticFaded committed:
example API working with gradio

guaneec committed:
Oops

C43H66N12O12S2 committed:
readd xformers attnblock

C43H66N12O12S2 committed:
Update sd_hijack.py

anastasiuspernat committed:
Added time left estimation

ArcticFaded committed:
prevent API from saving

C43H66N12O12S2 committed:
Update sd_hijack_optimizations.py

dfaker committed:
extras extend cache key with new upscale to options

JustMaier committed:
Auto select attention block for editing

dtlnor committed:
Create zh-hans.json

dtlnor committed:
Update zh-hans.json

dtlnor committed:
Update zh-hans.json

cvb011 commented:
I don't think localization can be done by simply replace words with it's meaning. It should be change more than that.

dtlnor committed:
Update zh-hans.json

dtlnor committed:
Update zh-hans.json

dtlnor committed:
Update zh-hans.json

dtlnor commented:
> I don't think localization can be done by simply replace words with it's meaning. It should be change more than that.

absolutely, therefore I am seeking help if anyone wants to improve my translation, expectually when some of the terms I am not famlilar to.

dtlnor committed:
Update zh-hans.json

cvb011 commented:
> > I don't think localization can be done by simply replace words with it's meaning. It should be change more than that.
> 
> absolutely, therefore I am seeking help if anyone wants to improve my translation, especially for the terms I am not famlilar to.

What I was trying to point out is, the localization we need is an accurate but in low degree of completion not those had a great completion. I think there is a lot of misleading transitions in you localization file, such as the word “upscaler” is more likely to be “上采样器” or “上采样算法” rather than “缩放算法”. 
Besides that I think it’s more suitable to leave those phrases alone rather than only translate half of them.
I’m currently working on this localization too, but due to academic reasons, my version won’t come up till tonight and I can’t translate those word which deeply linked to deep learning. Maybe you can refer some of my work and improve some of yours translations.

mykeehu committed:
Fix typo "celem_id" to "elem_id"

dtlnor commented:
> > > I don't think localization can be done by simply replace words with it's meaning. It should be change more than that.
> > 
> > 
> > absolutely, therefore I am seeking help if anyone wants to improve my translation, especially for the terms I am not famlilar to.
> 
> What I was trying to point out is, the localization we need is an accurate but in low degree of completion not those had a great completion. I think there is a lot of misleading transitions in you localization file, such as the word “upscaler” is more likely to be “上采样器” or “超分辨率算法” rather than “缩放算法”. Besides that I think it’s more suitable to leave those phrases alone rather than only translate half of them or use some kinds of mechanical translation tool. I’m currently working on this localization too, but due to academic reasons, my version won’t come up till tonight and I can’t translate those word which deeply linked to deep learning. Maybe you can refer some of my work and improve some of yours translations.

you are right, the term upscaler should point to 放大器/上采样器, rather than “缩放算法”, this is my mistake.
Besides, indeed some of the terms are not suitable to translate into another language. I will try to keep it as it should be.
Good to know you are also working on this, I will continue to update and improve this localization if this could help you a bit before you start your work. I will stop updating before night. If you think my version is okay but needs some polish, welcome to update my pr. If you want to do it completely by yourself and create a new pr, I am totally fine to close my pr and go to use your version straightly.

shwang95 commented:
Tiling is not 分块 but to generate images like tile laying patterns, I think 密铺 or 纹理 or something like that may be better.

ArcticFaded committed:
gradio and FastAPI

MalumaDev committed:
Merge branch 'master' into test_resolve_conflicts

dtlnor committed:
Update zh-hans.json

dtlnor commented:
> Tiling is not 分块 but to generate images like tile laying patterns, I think 密铺 or 纹理 or something like that may be better.

I see, seems I have some misunderstanding on tiling, I've corrected it to 密铺

cvb011 commented:
According to the comments in PR #3035 , I think we should slow our steps down a little bit to see how the community prefer to handle these localization files. How about a new repo to work out the best translation of the ui first?

fa0311 committed:
Add scripts to ui-config,json

dtlnor committed:
Update zh-hans.json

dtlnor commented:
> According to the comments in PR #3035 , I think we should slow our steps down a little bit to see how the community prefer to handle these localization files. How about a new repo to work out the best translation of the ui first?

ummmm, ~~I've just finish and did a rough proof read for the translation~~. Anyway, lets see how the community prefer the way they use localization files. If people want to keep translate out of the main repo, I will publish (or if anyone more familiar to this tool want to) a new repo for the best translation

cvb011 commented:
> > According to the comments in PR #3035 , I think we should slow our steps down a little bit to see how the community prefer to handle these localization files. How about a new repo to work out the best translation of the ui first?
> 
> ummmm, ~I've just finish and did a rough proof read for the translation~. Anyway, lets see how the community prefer the way they use localization files. If people want to keep translate out of the main repo, I will publish (or if anyone more familiar to this tool want to) a new repo for the best translation

Glad to see a great progress you made, and here is my current progress that you can take reffer of. cvb011/Stable-Diffusion-Webui-LocalizationFiles

ClashSAN committed:
Update README.md

fa0311 committed:
Remove exception handling

fa0311 committed:
Add visible for dropdown

fa0311 committed:
update scripts.py

AUTOMATIC1111 committed:
Merge branch 'master' into dark-mode

AUTOMATIC1111 committed:
Merge pull request #2939 from Michoko92/dark-mode

Added dark mode switch

AUTOMATIC1111 committed:
Merge pull request #2984 from guaneec/D

Don't eat colons in booru tags

AUTOMATIC1111 committed:
clarify the comment for the new option from #2959 and move it to UI section.

supersteve3d committed:
Update artists.csv

AUTOMATIC1111 committed:
remove shared option for update check (because it is not an argument of webui)
have launch.py examine both COMMANDLINE_ARGS as well as argv for its arguments

supersteve3d committed:
Update artists.csv

AUTOMATIC1111 committed:
fix for add difference model merging

AUTOMATIC1111 committed:
additional fix for difference model merging

AUTOMATIC1111 committed:
remove dependence on TQDM for sampler progress/interrupt functionality

liggest commented:
Hello, I also got a partial localized version at [here](https://github.com/liggest/stable-diffusion-webui-localization), maybe it could help.
#3073 seems also have one too.

MalumaDev committed:
Merge branch 'master' into test_resolve_conflicts

akemimadoka commented:
> Hello, I also got a partial localized version at [here](https://github.com/liggest/stable-diffusion-webui-localization), maybe it could help. #3073 seems also have one.

#3073 seems to be a result of machine translation

discus0434 committed:
add options to custom hypernetwork layer structure

discus0434 committed:
update

discus0434 committed:
update

discus0434 committed:
update

eltociear committed:
Fix typo in prompt_parser.py

assoicated -> associated

anastasiuspernat committed:
Estimated time displayed if jobs take more 60 sec

ArcticFaded committed:
provide sampler by name

anastasiuspernat committed:
Predictable long operation check for time estimation

realryo1 committed:
Fixed performance, vram style disorder

vacui-dev committed:
Update Learning Rate tooltip

chekaaa committed:
Remove wrong self reference in CUDA support for invokeai

dtlnor committed:
Update zh-hans.json

ArcticFaded committed:
call sampler by name

AUTOMATIC1111 committed:
more careful loading of model weights (eliminates some issues with checkpoints that have weird cond_stage_model layer names)

s-ilent committed:
Use training width/height when training hypernetworks.

AUTOMATIC1111 committed:
Merge remote-tracking branch 'origin/api'

discus0434 committed:
Merge branch 'master' into master

discus0434 committed:
Merge branch 'master' into master

AUTOMATIC1111 committed:
Merge branch 'master' into hot-reload-javascript

AUTOMATIC1111 committed:
Merge pull request #2835 from zhengxiaoyao0716/hot-reload-javascript

hot-reload javascript files

bgluminous commented:
> WIP, any update is welcome 抛砖引玉，只做了粗略的翻译，还需要精细打磨，欢迎纠错~~（我翻不了那么多，求大佬）~~ update: I've finished all the translation, welcome for any improvement

我已经完成了全部的翻译，希望这会有用
这是基于所有参数调研过的结果,这是所有参数的视频:
https://www.bilibili.com/video/BV1V8411s76T

I have finished all the translations. I hope it will be useful
This is based on the survey results of all parameters. This is the video of all parameters:
https://www.bilibili.com/video/BV1V8411s76T

Also See: #3119

MalumaDev committed:
Merge branch 'master' into test_resolve_conflicts

AUTOMATIC1111 committed:
remove the unneeded line break introduced by #3092

AUTOMATIC1111 committed:
fix for broken checkpoint merger

timntorres committed:
Infotext saves more specific hypernet name.

discus0434 committed:
layer options moves into create hnet ui

discus0434 committed:
enable to write layer structure of hn himself

AUTOMATIC1111 committed:
Merge pull request #3086 from discus0434/master

Add settings for multi-layer structure hypernetworks

discus0434 committed:
Merge branch 'AUTOMATIC1111:master' into master

AUTOMATIC1111 committed:
Merge pull request #3162 from discus0434/master

Added what I forgot to commit in already merged PR: #3086

dfaker committed:
Update ui.py

mezotaken committed:
Send empty prompts as valid generation parameter

AUTOMATIC1111 committed:
fix for #3086 failing to load any previous hypernet

asimard1 committed:
Update ui.py

discus0434 committed:
fix for #3086 failing to load any previous hypernet

discus0434 committed:
Merge branch 'AUTOMATIC1111:master' into master

asimard1 committed:
Added a bit of padding to the left

Mozoloa committed:
Fixed additionnnnal typo, sorry

fa0311 committed:
Add xformers message.

szhublox committed:
generation_parameters_copypaste.py: fix indent

dfaker committed:
update training header text

timntorres committed:
Issue #2921-Give PNG info to Hypernet previews.

timntorres committed:
Do nothing if image file already exists.

timntorres committed:
Revise comments.

dfaker committed:
update ui with extra training options

random-thoughtss committed:
Added support for RunwayML inpainting model

random-thoughtss committed:
Fixed copying mistake

random-thoughtss committed:
added support for ddim img2img

timntorres committed:
Remove redundant try/except.

random-thoughtss committed:
Don't compute latent mask if were not using it. Also added support for fixed highres_fix generation.

dfaker committed:
change html output

dfaker committed:
allow overwrite old hn

dfaker committed:
allow overwrite old embedding

dfaker committed:
split learn rates

dfaker committed:
pass overwrite old flag

dfaker committed:
reorder create_hypernetwork params

dfaker committed:
add existing caption file handling

dfaker committed:
link existing txt option

dfaker committed:
pass preprocess_txt_action param

dfaker committed:
default to ignore existing captions

discus0434 committed:
Merge branch 'AUTOMATIC1111:master' into master

discus0434 committed:
update

discus0434 committed:
update

discus0434 committed:
align ui.py imports with upstream

dfaker committed:
do caption copy for both flips

dtlnor committed:
Merge branch 'master' of https://github.com/dtlnor/stable-diffusion-webui

dtlnor committed:
Update zh-hans.json

dtlnor commented:
> 

thanks for your advice

random-thoughtss committed:
Fixed non-square highres fix generation

AUTOMATIC1111 committed:
allow float sizes for hypernet's layer_structure

AUTOMATIC1111 committed:
allow float sizes for hypernet's layer_structure

AUTOMATIC1111 committed:
Merge remote-tracking branch 'origin/master'

dtlnor committed:
Update zh-hans.json

Milly committed:
train: fixed preprocess image ratio

discus0434 committed:
Merge branch 'AUTOMATIC1111:master' into master

winterspringsummer committed:
Fixed path issue while extras batch processing

winterspringsummer committed:
Fixed path issue while extras batch processing

discus0434 committed:
add linear as a act func (option for doin nothing)

winterspringsummer committed:
Added try except to extras batch from directory

winterspringsummer committed:
Added try except to extras batch from directory

winterspringsummer committed:
Fixed path issue while extras batch processing

winterspringsummer committed:
Fixed path issue while extras batch processing

Milly committed:
train: ui: added `Split image threshold` and `Split image overlap ratio` to preprocess

guaneec committed:
Allow datasets with only 1 image in TI

wywywywy committed:
Implemented batch count logic to Outpainting mk2

wywywywy committed:
Implemented batch_size logic in outpainting_mk2

wywywywy committed:
Fixed outpainting_mk2 output cropping

sphuff committed:
[Bugfix][API] - Fix API arg in launch script

random-thoughtss committed:
Made dummy latents smaller. Minor code cleanups

Patryk27 committed:
interrogate: Fix CLIP-interrogation on CPU

Currently, trying to perform CLIP interrogation on a CPU fails, saying:

```
RuntimeError: "slow_conv2d_cpu" not implemented for 'Half'
```

This merge request fixes this issue by detecting whether the target
device is CPU and, if so, force-enabling `--no-half` and passing
`device="cpu"` to `clip.load()` (which then does some extra tricks to
ensure it works correctly on CPU).

mezotaken committed:
fix skip and interrupt for highres. fix option

random-thoughtss committed:
Added PLMS hijack and made sure to always replace methods

mezotaken committed:
Different approach to skip/interrupt with highres fix

random-thoughtss committed:
XY grid correctly re-assignes model when config changes

winterspringsummer committed:
sort file list in alphabetical ordering in extras

winterspringsummer committed:
sort file list in alphabetical ordering in extras

wywywywy committed:
Fixed job count & single-output grid

AUTOMATIC1111 committed:
updated readme and some small stylistic changes to code

AUTOMATIC1111 committed:
Merge remote-tracking branch 'origin/master'

AUTOMATIC1111 committed:
Merge pull request #3199 from discus0434/master

Add features to insert activation functions to hypernetworks

AUTOMATIC1111 committed:
a more strict check for activation type and a more reasonable check for type of layer in hypernets

AUTOMATIC1111 committed:
Merge branch 'master' into master

AUTOMATIC1111 committed:
Merge pull request #3263 from mezotaken/master

Fix skip and interrupt buttons for Highres. fix option enabled

AUTOMATIC1111 committed:
Merge branch 'master' into training-help-text

AUTOMATIC1111 committed:
Merge pull request #3197 from AUTOMATIC1111/training-help-text

Training UI Changes

AUTOMATIC1111 committed:
turns out LayerNorm also has weight and bias and needs to be pre-multiplied and trained for hypernets

AUTOMATIC1111 committed:
Fix Hypernet infotext string split bug for PR #3283

CCRcmcpe committed:
Change option description of unload_models_when_training

dtlnor committed:
Update zh-hans.json

Mozoloa committed:
wtf is happening

timntorres committed:
Sanitize hypernet name input.

timntorres committed:
Match hypernet name with filename in all cases.

timntorres committed:
Refer to Hypernet's name, sensibly, by its name variable.

timntorres committed:
Remove unused variable.

AUTOMATIC1111 committed:
Merge branch 'ae'

ClashSAN committed:
Update README.md

kybercore committed:
Make commit hash mandatory field

dfaker committed:
Use opts in textual_inversion image_embedding.py for dynamic fonts

AUTOMATIC1111 committed:
do not load aesthetic clip model until it's needed
add refresh button for aesthetic embeddings
add aesthetic params to images' infotext

AUTOMATIC1111 committed:
Merge remote-tracking branch 'origin/master'

AUTOMATIC1111 committed:
make aestetic embedding ciompatible with prompts longer than 75 tokens

dfaker committed:
Update image_embedding.py

AUTOMATIC1111 committed:
Merge branch 'master' into features-to-readme

AUTOMATIC1111 committed:
Merge pull request #3321 from AUTOMATIC1111/features-to-readme

Features to readme

AUTOMATIC1111 committed:
make aspect ratio overlay work regardless of selected localization

AUTOMATIC1111 committed:
Merge remote-tracking branch 'mk2/outpainting-mk2-batch-out'

AUTOMATIC1111 committed:
batch_size does not affect job count

AUTOMATIC1111 committed:
loading SD VAE, see PR #3303

AUTOMATIC1111 committed:
make aspect ratio overlay work regardless of selected localization, pt2

AUTOMATIC1111 committed:
Merge branch 'master' into fix/train-preprocess-keep-ratio

AUTOMATIC1111 committed:
Merge pull request #3246 from Milly/fix/train-preprocess-keep-ratio

Preprocess: fixed keep ratio and changed split behavior

AUTOMATIC1111 committed:
prevent error spam when processing images without txt files for captions

dtlnor committed:
Merge branch 'master' of https://github.com/dtlnor/stable-diffusion-webui

dtlnor committed:
Update zh-hans.json

dtlnor commented:
> Why the files changed tab is a complete mess? Which includes all the changes from original repo, hard to review.

Oh I guess I mess-up the git history, I will close this and re-open another pr with better source management
